bot/plotting/comparison_generator.py

```diff
@@ -0,0 +1,115 @@
+import pybamm
+import random
+from utils.chemistry_generator import chemistry_generator
+from plotting.plot_graph import plot_graph
+from experiment.experiment_generator import experiment_generator
+
+
+def comparison_generator(
+    number_of_comp,
+    models_for_comp,
+    chemistry,
+    provided_choice=None,
+):
+    params = pybamm.ParameterValues(chemistry=chemistry)
+    parameter_values_for_comp = dict(list(enumerate([params])))
+    comparison_dict = {}
+
+    if number_of_comp == 1:
+        param_list = []
+        diff_params = random.randint(2, 3)
+        for i in range(0, diff_params):
+            param_list.append(params.copy())
+            param_list[i][
+                "Current function [A]"
+            ] = chemistry_generator(
```

Comment:
Not for this issue, but in `chemistry_generator` can we just take the default value from the `chemistry` and then pick a number in some range based on that rather than having to specify an interval to pick from for each parameter set? I guess the range might be different for different parameters?

Reply:
Yes, I was about to update and rename it to generate values in the +- 10% (except for `"Current function [A]"`) range - 
```py
        params = pybamm.ParameterValues(chemistry=chemistry)
        param_value = desired_decimal_point_generator(
            params[parameter]*0.9, params[parameter]*1.1, 2
        )
```
I will push the code